packages/components/src/internal/url/URLResolver.ts

```diff
@@ -395,9 +400,14 @@ const LIST_MAPPERS = [
             const urlParts = parsePathName(row.get('url'));
 
             if (params && urlParts?.containerPath) {
-                const resolverPath = ListResolver.encodeResolverPath(urlParts.containerPath);
-                const parts = ['q', 'lists', resolverPath, params.listId];
-                return AppURL.create(...parts);
+                if (params.name) {
+                    const parts = ['q', 'lists', params.name];
+                    return AppURL.create(...parts);
```

Comment:
In the change above at line 385, the `params.name` if statement is before the `containerPath` check. This one is within. Does that matter here?

Reply:
Fixed.